templates/placementapi/config/placement.conf

```diff
@@ -1,6 +1,7 @@
 [DEFAULT]
 log_config_append = /etc/placement/logging.conf
 
```

Comment:
did you forget to add debug=true here

https://docs.openstack.org/placement/latest/configuration/config.html#DEFAULT.debug

Reply:
even more, i forgot to push chages with --force